kusama-runtime-constants/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "kusama-runtime-constants"
+version = "0.1.0"
+edition = "2018"
```

Comment:
look at the other cargo.toml files to see the metadata we expect

Reply:
fixed